joelethan/sendIT-react
# #165710463 Users should signup and login

### What does this PR do?
- This PR creates a signup page used by users to create an account
- This PR creates a login page used by users to Login

### Description of Task to be completed?
- Provide the user with a signup form
- Provide the user with a Login form
- Displays an error message in case of any

### How should this be manually tested?
- Clone the [repo](https://github.com/joelethan/sendIT-react) and cd into sendIT-react
- checkout the branch `ft-users-should-signup-and-login-165710463`
- Run the commands below to install packages and run the server
          - npm install
          - npm start
- Click the SignUp button and enter respective details to Register
- Click the Login button and enter respective details to Login

### What are the relevant pivotal tracker stories?

- [#165710463](https://www.pivotaltracker.com/story/show/165710463)

### Screenshots
Landing page
<img width="1188" alt="Screenshot 2019-05-02 at 11 18 17" src="https://user-images.githubusercontent.com/6951438/57063476-14d2d380-6ccc-11e9-92a2-042438e85fa3.png">
Signup page 
<img width="1188" alt="Screenshot 2019-05-02 at 11 19 12" src="https://user-images.githubusercontent.com/6951438/57063497-24521c80-6ccc-11e9-8a05-8dda6a25b819.png">
Login page
<img width="1188" alt="Screenshot 2019-05-02 at 11 19 36" src="https://user-images.githubusercontent.com/6951438/57063516-33d16580-6ccc-11e9-9b84-1b395e4e6185.png">
